webapp/package.json

```diff
@@ -19,59 +19,56 @@
     "start:local": "env-cmd -f ../.env next dev"
   },
   "dependencies": {
-    "@apollo/client": "^3.8.0-rc.1",
```

Comment:
there was a newer version, but anyway, `*-rc*` is not recommended to install them since it is a `release candite`, not ready for production environments